src/app/shared/components/capture-receipt/receipt-preview/receipt-preview.component.ts

```
@@ -119,25 +160,21 @@
       },
       cssClass: 'pop-up-in-center',
     });
-
     await closePopOver.present();
-
-    const { data } = await closePopOver.onWillDismiss();
-
+    const { data }: { data?: { action?: string } } = await closePopOver.onWillDismiss();
     if (data && data.action) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)